tutorials/peft-curation-with-sdg/main.py

```diff
@@ -245,43 +244,40 @@ def run_pipeline(args, jsonl_fp):
     # Disable synthetic data generation if the necessary arguments are not provided.
     if not args.synth_gen_endpoint:
         print(
-            "No synthetic data generation endpoint provided. Skipping synthetic data generation."
+            "No synthetic data generation endpoint provided. Skipping synthetic data generation.",
         )
         args.synth_gen_rounds = 0
     if not args.synth_gen_model:
         print(
-            "No synthetic data generation model provided. Skipping synthetic data generation."
+            "No synthetic data generation model provided. Skipping synthetic data generation.",
         )
         args.synth_gen_rounds = 0
     if not args.api_key:
         print(
-            "No synthetic data generation API key provided. Skipping synthetic data generation."
+            "No synthetic data generation API key provided. Skipping synthetic data generation.",
         )
         args.synth_gen_rounds = 0
 
     if args.synth_gen_rounds:
         print(
-            f"Using {args.synth_gen_endpoint}/{args.synth_gen_model} for synthetic data generation."
+            f"Using {args.synth_gen_endpoint}/{args.synth_gen_model} for synthetic data generation.",
         )
 
     synth_gen_ratio = args.synth_gen_ratio
     synth_gen_rounds = args.synth_gen_rounds
     synth_n_variants = args.synth_n_variants
 
-    assert (
-        synth_gen_ratio >= 0 and synth_gen_ratio <= 1
-    ), "The synthetic generation ratio must be between 0 and 1."
-    assert (
-        synth_gen_rounds >= 0
-    ), "The number of synthetic generation rounds must be a non-negative integer."
-    assert (
-        synth_n_variants >= 1
-    ), "The number of synthetic variants must be a positive integer."
+    if synth_gen_ratio >= 0 and synth_gen_ratio <= 1:
+        msg = "The synthetic generation ratio must be between 0 and 1."
+        raise ValueError(msg)
+    elif synth_gen_rounds >= 0:
+        msg = "The number of synthetic generation rounds must be a non-negative integer."
+        raise ValueError(msg)
+    elif synth_n_variants >= 1:
+        msg = "The number of synthetic variants must be a positive integer."
+        raise ValueError(msg)
```

Comment:
It looks like these error checks are formatted the opposite way?

Reply:
OOPS! Great catch